core/web/template_engine.ex

```diff
@@ -5,46 +5,61 @@ defmodule AntikytheraCore.TemplateEngine do
   This is an implementation of `EEx.Engine` that auto-escape dynamic parts within HAML templates.
   """
 
-  use EEx.Engine
-  alias Antikythera.TemplateSanitizer
+  @behaviour EEx.Engine
 
-  def init(_opts), do: {:safe, ""}
+  alias Antikythera.TemplateSanitizer
 
-  def handle_body({:safe, iodata}) do
-    q =
-      quote do
-        IO.iodata_to_binary(unquote(iodata))
-      end
-    {:safe, q}
+  @impl true
+  def init(_opts) do
+    %{
+      iodata:    [],
+      dynamic:   [],
```

Comment:
The above line is the same.
```suggestion
      dynamic:    [],
```

Reply:
fixed in https://github.com/access-company/antikythera/pull/79/commits/8db8d5dd046ba0969393479b533730fa335bb79a